### ISSUE TITLE
Image instability 

### ISSUE THREAD

**Issue by GitRayc** [helgeerbe/picframe#346]
Having some image instability issues on raspberrypi bookworm.

Tried several test to determine what could be happening so I decided to work with only 7 images to make is simple.

5 are png files
2 are jpg images        all will display correctly with no problems on both mac os and windows 10

They are reside on a USB drive that mounts on boot
Originally all were jpg files and would display.
I exported 5 of them as png files

Suddenly all the png files started to display and blend in and our beautifully.  BUT.......  
this only lasted for a while.

After a reboot I now have only two of the images that will play in the slide show.  And those are two of the 5 png files that were all displaying previously. None of jpg images will play

All the jpg images that I had on my raspberry pi 3 with old OS will still play on the pi 4

Anyone have any ideas.   I love picframe and all that everyone does to support it.   But unfortunately my new install of pi4 and bookworm is simply an exercise in frustration.

Anyone have any ideas?

**Comment by GitRayc** [comment]
Please forgive me.  I fear I might be abusing this board.  Sorry
So further investigation; it seems to be a USB drive problem.

I copied all the same pics to the Picture directory in /home/pi and every single one displays as it should. And of course edited yaml file to look at "Pictures" rather then "USBPICS"

The USB does mount automatically, I can access it and view all the pic files on it. But problems with displaying them as I mentioned.   Anyone know why this could be a problem.  I tried two different USB drives formatted as FAT32.  Both are 16 gig USB thumb drives

**Comment by GitRayc** [comment]
Well I thought this was the end of it,  But no!   The images loaded into the pictures directory do display, but after displaying a few of them an image just hangs and the blend will no longer continue to blend to the next image.   And this hang is not on the same image each time!

**Comment by GitRayc** [comment]
Update:
Well after driving myself crazy I gave up and closed my Mac.   
 
And all of a sudden all the image display problems and freezing went away.
Using my test USB Thumbdrive with the pics I mentioned suddenly worked peferctly.  Ran through a dozen sequences of blend with no problems all at.

I dont understand at all but maybe someone does.  It seems that if any connection is open between my mac and pi then I have this pic problems.   Seems with Mac shut down all is fine.
